shaders/src/shading_lit.fs

```diff
@@ -134,9 +174,11 @@ void getRoughnessPixelParams(const MaterialInputs material, inout PixelParams pi
     perceptualRoughness = mix(perceptualRoughness, basePerceptualRoughness, pixel.clearCoat);
 #endif
 
+    pixel.perceptualRoughnessUnclamped = perceptualRoughness;
+    // Clamp the roughness to a minimum value to avoid divisions by 0 during lighting
+    pixel.perceptualRoughness = clamp(perceptualRoughness, MIN_PERCEPTUAL_ROUGHNESS, 1.0);
```

Comment:
Is this correct to clamp after normal filtering?  Not sure it is.

Reply:
I wasn't sure either, but looking at the code, there shouldn't be any over/under flow, AFAIK (i.e. that code doesn't have any divides, or other ops that could cause problems). So I thought that our clamping should be done as late as possible anyways, since it's just for avoiding div0.

Reply:
That code increases roughness for anti-aliasing based on variance. It's be interesting to compare clamping before and after. Probably doesn't make a big difference.

Reply:
I mean if we didn't have the div0 problem, we wouldn't clamp here right? so that code should be running "as if" we didn't have the problem.
Did we do the clamping before filtering on purpose or did it just happen this way?

If we don't do that, then I would have to do the math twice, which seems a waste.

Reply:
Why would we do the math twice? I think it's fine to clamp after, I just wanted to make sure we were not missing something.